server/api/user/user.model.js

```diff
@@ -18,9 +18,9 @@ var UserSchema = new Schema({
   hashedPassword: String,
   provider: String,
   salt: String,
-  tech: [String],
+  tech: [String], 
   projects: [{type : Schema.Types.ObjectId, ref: 'Project'}], // project id
-  bio:String,
+  bio: {type:String, default: "No Bio written yet. Please tell us some more about yourself"},
```

Comment:
I think it's much better to keep the check client side. That way we know
easily who hasn't update there info, all we have to do is check for an
empty string instead of querying/checking for a default string.

On Thu, Sep 24, 2015 at 3:19 PM, Severin Ibarluzea <notifications@github.com
> wrote:

> In server/api/user/user.model.js
> <https://github.com/rcos/Observatory3/pull/151#discussion_r40360697>:
>
> >    projects: [{type : Schema.Types.ObjectId, ref: 'Project'}], // project id
> > -  bio:String,
> > +  bio: {type:String, default: "No Bio written yet. Please tell us some more about yourself"},
>
> @agundy <https://github.com/agundy> should we do this client side or
> server side? My intuition says it might be better on the client, but this
> does solve the problem.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rcos/Observatory3/pull/151/files#r40360697>.
>